dch/dch.js

```diff
@@ -1,86 +1,67 @@
 'use strict';
 
-// These are the modules imported from 'node_modules' directory. Couldn't get
-// the Airbnb style of "import { express } from 'express';" working.
+var database = require('./DB.js');
+var mqttSubscriber = require('./mqtt-subscriber/mqtt-subscriber.js');
+
 const express = require('express');
-const mongoose = require('mongoose');
-const MongoClient = require('mongodb').MongoClient;
-let mqttSubscriber = require('./mqtt-subscriber/mqtt-subscriber.js');
-//const mqttSubscriber = require('./mqtt-subscriber.js');
+const app = express();
+const router = express.Router();
 
-// Port to run app
-const PORT_APP = 3000;
+const APP_PORT = 3000; 
 
-// This is the path to the application index and static root directory 
-const PUBLIC_DIRECTORY = 'public';
 
-// API paths
+//basic paths 
 const NO_PATH = '/';
-const GET_DATA_PATH = '/get_data/';
-const DELETE_DATA_PATH = '/delete_data/';
-const TOTAL_CONNECTED_DEVICES_PATH = '/total_connected_devices/';
-
-// URLs
-const MONGODB_URL = 'mongodb://localhost:27017/mqtt-database';
-
-// Mongoose events
-const MONGOOSE_OPEN = 'open';
-
-// Mongoose Schema data. Didn't finish reading up on the intricacies of
-// Mongoose. This can be made more elaborate but it is enough to fetch all
-// data required as a single JSON object.
-const acccelerationCollectionDataTypes = {data: String};
-const accelerationCollection = {collection: 'mqtt-data'};
-const watchDataModelName = 'watchDataModel';
-
-// Conosle messages and errors
-const SERVER_INITIALIZED_MESSAGE = 'View page in browser at ' +
-  '"http://localhost:3000"';
-const DB_CONNECTION_ESTABLISHED_MESSAGE = 'Connection made with MongoDB';
-
-const TOTAL_NODEJS_MQTT_CLIENTS = 2;
-
-// Database schema and connection configurations
-let watchDataSchema = mongoose.Schema(
-  acccelerationCollectionDataTypes,
-  accelerationCollection
-);
-
-let watchDataModel = mongoose.model(watchDataModelName, watchDataSchema);
-let db = mongoose.connection;
-
-let app = express();
-let systemListener = mqttSubscriber.sysClient;
+const API_BASE_PATH = '/api';
+const PUBLIC_DIR = 'public';
+//API paths
+const DELETE_DATA_PATH = '/deleteData/:watchId';
```

Comment:
URL style deviates from convention. Words should be seperated by hyphens. Not camel case (or underscores for that matter...)

Good material on it here:

http://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris